pkg/resourcemanager/vm/reconcile.go

```diff
@@ -91,10 +101,19 @@ func (g *AzureVirtualMachineClient) Ensure(ctx context.Context, obj runtime.Obje
 			switch azerr.Type {
 			case errhelp.AsyncOpIncompleteError:
 				instance.Status.Provisioning = true
+				instance.Status.Message = err.Error()
```

Comment:
This appears to already be set at the top of the error block on line 86

Reply:
removed, thanks for pointing out.